UPDATING_BADGES_AND_PR_CHECKS.md

```diff
@@ -0,0 +1,120 @@
+do), so they "automagically" work when your repo is created.
+
+Unfortunately the other badges are going from sources outside of GitHub,
+so we can't use the same trick for them.
+
+### BetterCodeHub and LGTM badges
+
+When GitHub Classroom makes the initial copies of this repository, the
+[BetterCodeHub](https://bettercodehub.com) (BCH) and
+[LGTM](https://lgtm.com) ("looks good to me") badges have several copies
+of this string:
+
+```html
+UMM-CSci-3601/3601-iteration-template
+```
+
+This tells those services to include the badges for the `3601-iteration-template` repository in the `UMM-CSci-3601` organization. You need to update both the organization and the repository name so they match the "name" of your repository up
+at the top of your repository page.
+
+* The organization should be changed to whatever the GitHub
+  organization is for this semester's course. This is probably
+  something like `UMM-CSci-3601-S21`.
+* The repository is whatever name you've given your repository,
+  like `it-1-stupendous-wombats`.
+
+You then need to change all the instances of
+
+```html
+UMM-CSci-3601/3601-iteration-template
+```
+
+so they match the name of your repository, e.g.,
+
+```html
+UMM-CSci-3601-S21/it-1-stupendous-wombats
+```
+
+Once all those are updated, then the badges should point to the
+correct analysis and reflect the state of _your_ project. These don't
+always update immediately because of browser caching, but if you click
+on the badges that should take you to pages on that analysis service that
+reflect _your_ project.
+
+## Enabling pull request checks
+
+Below are instructions for enabling pull request checks from
+the external services so that your badges will update and
+"mean something".
+
+### BetterCodeHub
+
+To enable pull request checks from BetterCodeHub you need to:
+
+1. Go to [BetterCodeHub](https://bettercodehub.com) and login using
+   your GitHub account.
+2. Choose this semester's organization icon from the set of icons to the
+   right of "Your repositories".
+3. That should display all the repositories for this semester, which may
+   be rather a lot towards the end. Scroll or search to find your
+   specific repository.
+4. On the bottom left of your repo's card should be the GitHub pull
+   request icon. Click that to enable BetterCodeHub pull request checks
+   for that repository.
+
+You should then see BetterCodeHub checks on your next pull request on
+that project.
+
+### LGTM
+
+To enable pull request checks from LGTM you need to:
+
+1. Go to [LGTM](https://lgtm.com) and login using your GitHub account.
+2. Click "Project lists" in the upper right.
+3. Type/paste the _full repo URL_ into the "Follow a project from a
+   repository host" search bar and press "Follow". That should add your
+   repository to the list of repositories that you're following.
+4. Click the big green button to the right of the listing for your
+   repository to enabled automated PR reviews.
+
```

Comment:
Should we suggest that students can disable the automatic commenting that LGTM does by default?

Reply:
Good question. I probably should add a note about that.